doc/api/crypto.md

```diff
@@ -1886,10 +1854,15 @@ added: v0.1.92
 * `options` {Object} [`stream.Writable` options][]
 * Returns: {Sign}
 
-Creates and returns a `Sign` object that uses the given `algorithm`.
-Use [`crypto.getHashes()`][] to obtain an array of names of the available
-signing algorithms. Optional `options` argument controls the
-`stream.Writable` behavior.
+Creates and returns a `Sign` object that uses the given `algorithm`.  Use
+[`crypto.getHashes()`][] to obtain the names of the available digest algorithms.
+Optional `options` argument controls the `stream.Writable` behavior.
+
+In some cases, a `Sign` instance can be created using the name of a signature
+algorithm, such as 'RSA-SHA256', instead of a digest algorithm. This will use
```

Comment:
Not sure about these four suggestions, but to be on the safe side:
```suggestion
algorithm, such as `'RSA-SHA256'`, instead of a digest algorithm. This will use
```